lib/src/switch/Switch.tsx

```diff
@@ -125,65 +110,53 @@ const calculateWidth = (margin, size) =>
     ? `calc(${sizes[size]} - ${getMargin(margin, "left")} - ${getMargin(margin, "right")})`
     : sizes[size];
 
-const getDisabledColor = (props, element, subelement?) => {
+const getDisabledColor = (
+  theme: AdvancedTheme["switch"],
+  element: "track" | "thumb" | "label",
+  subElement?: "check" | "uncheck"
+) => {
   switch (element) {
     case "track":
-      switch (subelement) {
+      switch (subElement) {
```

Comment:
Subelement is a single word.

Reply:
**subelement** is not a word, is **sub-element**.